### PR TITLE
Increased the stability of the task completion and display process

### PR DESCRIPTION
**Pull request by Tomentosa**
Deleted un used files, adjusted TaskTable Scaffold to use flex widgets that played nicely with many items Got up to 21 items
CleanUp of dead code relating to tasks across the repo
re-factored tasks files to eliminate reducnancy/double calls

### CODE REVIEW

**Comment by Tomentosa** [comment]
Hey @Daryle, @JoshuaDFischer is gonna check that he's cool w this branch on Monday. Don't merge until he gives the thumbs upp

File: lib/Screens/Tasks/completeActivity.dart [hunk 1]
```diff
@@ -35,8 +35,13 @@ class _TaskDetails extends State<TaskDetails> {
...
     final taskObserver = Provider.of<TaskObserver>(context, listen: false);
 
-    responseText = taskObserver.currTaskForDetails!.responseText;
-    print('response text = ' + responseText);
+
+    try {
+      responseText = taskObserver.currTaskForDetails!.responseText;
+      print('response text = ' + responseText);
+    } on Exception catch (e) {
+      print('Exception $e');
```
Comment (Tomentosa) [inline lib/Screens/Tasks/completeActivity.dart:43]: We could have this exception take you back to the Tasks screen instead. This popped up during testing so I surrounded it with a try catch

File: lib/Screens/Tasks/completeHealthCheck.dart [hunk 2]
```diff
@@ -683,7 +707,15 @@ class _TaskHealthCheck extends State<TaskHealthCheck> {
                                     style: TextStyle(fontSize: 20),
                                   ),
                                   onPressed: () {
-                                    _onSave(taskObserver);
```
Comment (Tomentosa) [inline lib/Screens/Tasks/completeHealthCheck.dart:686]: Passing in the task observer was giving weird results, so I just did those operations right in the on pressed method instead

**Comment by Daryle** [comment]
> Hey @Daryle, @JoshuaDFischer is gonna check that he's cool w this branch on Monday. Don't merge until he gives the thumbs upp

I got u, Thanks for the heads up!

**Pull request closed without merging**